br/pkg/task/common.go

```diff
@@ -201,6 +206,8 @@ func DefineCommonFlags(flags *pflag.FlagSet) {
 
 	flags.Bool(flagCheckRequirement, true,
 		"Whether start version check before execute command")
+	flags.Bool(flagSkipCollactionCheck, false,
```

Comment:
Maybe move it to `DefineRestoreFlags`? This flag seems useless when doing backup.